.ci/azure/linux.yml

```diff
@@ -49,3 +51,29 @@ jobs:
     submodules: recursive
     path: openvino_contrib
 
+  - checkout: git://openvinotoolkit/openvino@master
+    clean: true
+    lfs: false
+    submodules: recursive
+    path: $(OPENVINO_REPO_DIR)
+
+  - script: |
+      sudo apt --assume-yes install libusb-1.0-0-dev
+    displayName: 'Install dependencies'
+
+  - script: |
+      wget https://github.com/ninja-build/ninja/releases/download/v1.10.0/ninja-linux.zip
+      unzip ninja-linux.zip
+      sudo cp -v ninja /usr/local/bin/
+    workingDirectory: $(WORK_DIR)
+    displayName: 'Install Ninja'
+
+  - task: CMake@1
+    inputs:
+      cmakeArgs: -GNinja -DVERBOSE_BUILD=ON -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) $(OPENVINO_REPO_DIR)
+      workingDirectory: $(BUILD_OPENVINO_DIR)
+
+  - script: ninja
+    workingDirectory: $(BUILD_OPENVINO_DIR)
+    displayName: 'Build OpenVINO'
+

```

Comment:
we also need to pass `-DIE_EXTRA_MODULES=openvino_contrib` somewhere here

Reply:
Yes, after adding Java bindings here.